lib/module_importer.js

```diff
@@ -105,24 +103,15 @@ function makeImporter(eyeglass, sass, options, fallbackImporter) {
 
   return function(uri, prev, done) {
     var isRealFile = fs.existsSync(prev);
-    var match = uri.match(IMPORT_REGEX);
-
-    if (match) {
-      // This is an import of the form "<node_module>/some/file"
-      var moduleName = match[1];
-      var relativePath = match[2];
-      var pkgRootDir = isRealFile ? packageRootDir(path.dirname(prev)) : root;
-      var jsFile = getModuleByName(moduleName, pkgRootDir);
-
-      if (!jsFile) {
-        console.error("No Eyeglass module named '" + moduleName +
-                      "' could be found while importing '" + uri + "'.");
-        done({});
-        return;
-      }
-
+    var fragments = uri.split("/");
+    var moduleName = fragments[0];
+    var relativePath = fragments.slice(1).join("/");
+    var pkgRootDir = isRealFile ? packageRootDir(path.dirname(prev)) : root;
+    var jsFile = getModuleByName(moduleName, pkgRootDir);
+
+    if (jsFile) {
```

Comment:
By testing positive for `jsFile` instead of negative, we can slip through to the `isRealFile` path or the native Sass importer.